app/assets/stylesheets/simulator.scss

```diff
@@ -1,3 +1,30 @@
 // Place all the styles related to the simulator controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+$green: #42b983;
+$teal: #026e57;
+$gray-100: #f9fffc;
+$gray-dark: #b9b9b9;
+$blue: #337ab7;
+$primary: $gray-dark;
+$secondary: $blue;
+$border-radius: .3rem;
+$font-familiy-sans-serif: 'Open Sans', 'Noto Sans';
+$input-btn-focus-width: .2rem;
+$color-hover: $green;
+$active-background: #fff;
+$active-border: $teal;
```

Comment:
Many of these styles are either not needed or default, just use variables which are needed for simulator layout